src/nls/root/strings.js

```diff
@@ -601,7 +602,8 @@ define(function (require, exports, module) {
                 RECENT_DOC: "Recent Documents",
                 DOCUMENT: "Documents",
                 MENU_COMMAND: "Menu Commands",
-                LAYER: "Layers",
+                ALL_LAYER: "Layers",
+                CURRENT_LAYER: "Layers",
```

Comment:
I wonder if there is some way we can distinguish between these two categories that is helpful but not overly verbose....